src/transitions/scroll-transition.js

```diff
@@ -83,20 +103,25 @@ export default class ScrollTransition extends React.Component {
   getSlideStyles(index, positionValue) {
     const targetPosition = this.getSlideTargetPosition(index, positionValue);
     return {
-      position: 'absolute',
-      left: this.props.vertical ? 0 : targetPosition,
-      top: this.props.vertical ? targetPosition : 0,
+      boxSizing: 'border-box',
       display: this.props.vertical ? 'block' : 'inline-block',
-      listStyleType: 'none',
-      verticalAlign: 'top',
-      width: this.props.vertical ? '100%' : this.props.slideWidth,
       height: 'auto',
-      boxSizing: 'border-box',
-      MozBoxSizing: 'border-box',
+      left: this.props.vertical ? 0 : targetPosition,
+      listStyleType: 'none',
+      marginBottom: this.props.vertical ? this.props.cellSpacing / 2 : 'auto',
       marginLeft: this.props.vertical ? 'auto' : this.props.cellSpacing / 2,
       marginRight: this.props.vertical ? 'auto' : this.props.cellSpacing / 2,
       marginTop: this.props.vertical ? this.props.cellSpacing / 2 : 'auto',
-      marginBottom: this.props.vertical ? this.props.cellSpacing / 2 : 'auto'
+      MozBoxSizing: 'border-box',
+      position: 'absolute',
+      top: this.props.vertical ? targetPosition : 0,
+      transform:
+        this.props.animation === 'zoom' && this.props.currentSlide !== index
+          ? 'scale(0.85)'
```

Comment:
I imagine we'll want to make this scale configurable via a prop in the future. Maybe create an issue and add a  comment here?

Reply:
good idea